server/public/styles.css

```diff
@@ -440,6 +445,85 @@ input::-webkit-inner-spin-button {
   float: left;
 }
 
+.add-comment-button {
+  border-radius: 5px;
+  margin: 20px 0;
+  margin-bottom: 20px;
+  padding: 0 1rem;
+  text-align: center;
+  background-color: rgb(240, 131, 52);
+  color: white;
+  border: none;
+  height: 2rem;
+  letter-spacing: 2px;
+  font-size: 1.3rem;
+  font-weight: 700;
+  cursor: pointer;
+  float: right;
+}
```

Comment:
I'm seeing a lot of duplication in your CSS for things like buttons. Try making a `.app-button` class that combines all of the common properties like `border`, `letter-spacing`, `cursor`, and `color` (and all of the other duplicates) into one place. Then use that in your elements along with the special classes. Try to see how much extra CSS code you can eliminate, by examining your other rules too.